src/inc/cor.h

```diff
@@ -2195,8 +2195,12 @@ inline ULONG CorSigUncompressData(      // return number of bytes of that compre
 
 
 #if !defined(SELECTANY)
+#if defined(__GNUC__) && !defined(__llvm__)
+    #define SELECTANY extern __attribute__((selectany))
```

Comment:
Did some searching and seems like LLVM's own codebase and other folks in FOSS are using `__attribute__((weak))`, in case of `defined __GNIC__`, where the else case has `__declspec(selectany)`.

Reply:
Makes sense. I also found a weak declaration inside the pal layer. Replaced to match.